libevm/VMValidate.cpp

```diff
@@ -36,22 +40,23 @@ void VM::validate(ExtVMFace& _ext)
 	m_ext = &_ext;
 	initEntry();
 	size_t PC;
-	byte OP;
-	for (PC = 0; (OP = m_code[PC]); ++PC)
-		if  (OP == byte(Instruction::BEGINSUB))
+	Instruction OP;
```

Comment:
Are the changes in this file supposed to go into this PR ? They look not really related to SIMD